src/main/java/gomedic/model/person/AbstractPerson.java

```diff
@@ -69,7 +67,8 @@ public boolean equals(Object other) {
 
         AbstractPerson otherPerson = (AbstractPerson) other;
         return isSamePerson(otherPerson)
-                && otherPerson.getId().equals(getId());
+                && otherPerson.getName().equals(name)
```

Comment:
hem im slightly confused why are you still comparing name and phone here

Reply:
Yup sure, I think the `AB3` doesn't have ID and to be honest I don't find to have two equality comparator to be useful haha